packages/manager/src/features/CloudPulse/Alerts/constants.ts

```diff
@@ -0,0 +1,13 @@
+import type { AlertSeverityType } from '@linode/api-v4';
+
+export interface Item<T = number | string, L = string> {
+  data?: any;
+  label: L;
+  value: T;
+}
+export const alertSeverityOptions: Item<AlertSeverityType>[] = [
+  { label: 'Info', value: 3 },
+  { label: 'Low', value: 2 },
+  { label: 'Medium', value: 1 },
+  { label: 'Severe', value: 0 },
+];
```

Comment:
I'd prefer we don't use that generic `Item` type. It has that extra `data` property that is not typesafe and does not seem to be needed anyway.

I recommend we just use a simple one-off type like this:

```suggestion
interface AlertSeverityOption {
  label: string;
  value: AlertSeverityType;
}

export const alertSeverityOptions: AlertSeverityOption[] = [
  { label: 'Info', value: 3 },
  { label: 'Low', value: 2 },
  { label: 'Medium', value: 1 },
  { label: 'Severe', value: 0 },
];
```

Reply:
After discussing, I think we should either go with 

```typescript
interface AlertSeverityOption {
  label: string;
  value: AlertSeverityType;
}
```

or something like

```typescript
export interface Item<L extends string, T> {
  label: L;
  value: T;
}

export const alertSeverityOptions: Item<string, AlertSeverityType>[] = [
  { label: 'Info', value: 3 },
  { label: 'Low', value: 2 },
  { label: 'Medium', value: 1 },
  { label: 'Severe', value: 0 },
];
```